include/gridtools/stencil/naive.hpp

```diff
@@ -39,11 +39,10 @@ namespace gridtools {
                     auto extent = info.extent();
                     auto interval = stages_t::interval();
                     auto num_colors = info.num_colors();
-                    auto offsets =
-                        tuple_util::make<hymap::keys<dim::i, dim::j, dim::k>::values>(-extent.minus(dim::i()),
-                            -extent.minus(dim::j()),
-                            -grid.k_start(interval) - extent.minus(dim::k()));
-                    auto sizes = tuple_util::make<hymap::keys<dim::c, dim::k, dim::j, dim::i>::values>(
+                    auto offsets = hymap::keys<dim::i, dim::j, dim::k>::make_values(-extent.minus(dim::i()),
```

Comment:
is `make_values` needed as it is host code?

Reply:
yes. `nvcc` fails here even it the code is host only

Reply:
For the case when you compile the naive backend with nvcc? Then what about `cpu_ifirst/entry_point.hpp` where you use just `make`.